src/main.ts

```diff
@@ -61,6 +61,7 @@ const wait_for_input = () => {
       student_id: idNum
     }, 1)
   })
+  console.log(getmac());
```

Comment:
This will console log it every time that we have a new request... Keep in mind that you can put things at the top level of the file, you don't have to put it in a function. I would recommend just putting this console log literally right after the `getmac` decl